presto-verifier/src/main/java/com/facebook/presto/verifier/rewrite/QueryRewriter.java

```diff
@@ -113,15 +113,15 @@ public QueryRewriter(
             PrestoAction prestoAction,
             Map<ClusterType, QualifiedName> tablePrefixes,
             Map<ClusterType, List<Property>> tableProperties,
-            Optional<String> nonDeterministicFunctionSubstitutes)
+            Optional<String> functionSubstitutes)
```

Comment:
Similar comment here. Should the parameter be a `Multimap`?

Reply:
Revised